packages/SwingSet/src/kernel/vat-loader/manager-subprocess-xsnap.js

```diff
@@ -112,10 +112,10 @@ export function makeXsSubprocessFactory({
     }
 
     const vatKeeper = kernelKeeper.provideVatKeeper(vatID);
-    const lastSnapshot = vatKeeper.getLastSnapshot();
-    if (lastSnapshot) {
-      const { snapshotID } = lastSnapshot;
-      kernelSlog.write({ type: 'heap-snapshot-load', vatID, snapshotID });
+    const snapshotInfo = vatKeeper.getSnapshotInfo();
+    if (snapshotInfo) {
+      const { hash, endPos } = snapshotInfo;
+      kernelSlog.write({ type: 'heap-snapshot-load', vatID, hash, endPos });
```

Comment:
I think this slog entry uses `hash`, while the `heap-snapshot-save` one uses `snapshotID`.. we need to make them consistent. I think I prefer using `snapshotID` rather than `hash`. You might consider making the contents of `snapshotInfo` use `snapshotID` instead of `hash` (and avoiding variables named `hash`) to make that consistency easier.

Reply:
I quite prefer `hash`, since really there's no longer any such thing as a "snapshot ID".  The hash is not being used to identify the snapshot but is instead being used more like a checksum to validate that it consists of the bits that it's supposed to consist of.  However, being consistent seems right; I think the proper thing is to excise `snapshotID` from the few remaining places it appears.  Having 'hash' in the slog entry says "I saved/loaded a snapshot and its hash was blah" - i.e., the hash is now just more descriptive metadata.